src/query_parser/v2/lexer.rl

```diff
@@ -25,6 +25,67 @@ void RSQuery_Parse_v2(void *yyp, int yymajor, QueryToken yyminor, QueryParseCtx
 void *RSQuery_ParseAlloc_v2(void *(*mallocProc)(size_t));
 void RSQuery_ParseFree_v2(void *p, void (*freeProc)(void *));
 
+int RSQuery_ParseNumericOp_v2(void* pParser, int OperatorType, QueryToken tok, 
+      QueryParseCtx *q, const char *ts, const char *te, char c1,
+      unsigned int opLen) {
+    tok.len = te - (ts + 1);
+    int len = tok.len;
+    tok.pos = ts - q->raw;
+    tok.s = ts + 1;
+
+    // Find the position before the operator
+    char *end1 = strchr(tok.s, c1) - 1;
+    // Find the position after the operator
+    char *start2 = end1 + opLen + 1;
+    // Remove spaces before the operator
+    while (isspace(*end1)) {
```

Comment:
Can a modifier have escaped spaces? e.g.,
"@mod\\\\ >42"

Reply:
Yes. Supported by [afb43ad](https://github.com/RediSearch/RediSearch/pull/4772/commits/afb43ad06a44c535a602164d58f4bbdeac7be9ae).  Tests in `test_numbers.py` : `testNumericOperatorsModifierWithEscapes()`